project/index.mjs

```diff
@@ -433,6 +433,34 @@ router.post('/:projectId/removeCustomRoles', auth0Middleware(), async (req, res)
     respondWithError(res, error.status ?? 500, error.message ?? 'Error removing custom roles.')
   }
 })
+ 
+// Update Project Metadata
+router.route("/:projectId/update-metadata").put(auth0Middleware(), async (req, res) => {
```

Comment:
Simply a `PUT :id/metadata` would imply it is being updated, I think. The route may be overnamed.

Reply:
Just try to be consistent and sane.  Compare against the other .put()s.
![image](https://github.com/user-attachments/assets/9332d6bb-44a2-4bb0-955f-2a9fb62aeb87)